package.json

```diff
@@ -24,13 +23,15 @@
     "karma-browserify": "^4.2.1",
     "karma-chrome-launcher": "~0.1.2",
     "karma-jasmine": "^0.2.2",
-    "karma-phantomjs-launcher": "~0.1.2"
+    "karma-phantomjs-launcher": "~0.1.2",
+    "react": "^15.1.0",
```

Comment:
added react because its peer dependency for reflux 0.4.x (dont know if this can be avoided)